Die Umstellung auf von uclibc auf musl benoetigt einen olsrd-Patch, der erst nach v0.6.8
eingefuehrt wurde:
 http://comments.gmane.org/gmane.network.olsr.devel/7592

Leider wurde der Branch "for-15.05" noch nicht auf v0.9 aktualisiert.
Der untenstehende Patch behebt dies.

--- a/routing/olsrd/Makefile	2015-06-24 22:42:22.843154799 +0200
+++ b/routing/olsrd/Makefile	2015-06-24 22:42:38.055454470 +0200
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=olsrd
-PKG_VERSION:=0.6.8
-PKG_RELEASE:=2
+PKG_VERSION:=0.9.0.1
+PKG_RELEASE:=4
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=http://www.olsr.org/releases/0.6
+PKG_SOURCE_URL:=http://www.olsr.org/releases/0.9
 
-PKG_MD5SUM:=feabdd611391dcb30af5795e834cc258
+PKG_MD5SUM:=5dec2c08e7bbec6789af7279e7aafc6c
 PKG_BUILD_PARALLEL:=1
 PKG_LICENSE:=BSD-3-Clause
 
